qdrant-landing/content/documentation/concepts/storage.md

```diff
@@ -7,19 +7,23 @@ aliases:
 
 # Storage
 
-All data within one collection is divided into segments.
-Each segment has its independent vector and payload storage as well as indexes.
+In a Qdrant collection, all data is divided into **segments**. Each segment contains several components: **vector** and **payload storages** as well as **vector** and **payload indexes**. Behind the scenes, each segment has an ID mapper that links internal and external IDs. 
```

Comment:
Also here, there's a layer in between.

Collections have shards. There may be multiple copies of shards, which we call replicas. Shards have segments.

We normally mostly abstract this concept away, and segments aren't visible on the logical data level.

I understand we might want to describe segments somewhere, but describing it in the [optimizer](https://qdrant.tech/documentation/concepts/optimizer/) section may be better. They're closely related to the optimization process.

---

```suggestion
In a Qdrant shard, all data is divided into **segments**. Each segment contains several components: **vector** and **payload storages** as well as **vector** and **payload indexes**. Behind the scenes, each segment has an ID mapper that links internal and external IDs. 
```